src/lambda_test_notification_service.py

```diff
@@ -66,12 +57,20 @@ def lambda_handler(event, context):
 
     message = Message(formatted_message, message_subject)
 
+    secret_client = SecretManager(region_name="eu-north-1")
```

Comment:
1. Please don't use region other than eu-central-1
2. resources created in eu-north-1 and other regions - should be deleted
3. I guess, here we don't need parameter "region_name" - just read secret from default region